src/app.py

```diff
@@ -19,50 +21,77 @@ def main():
     )
 
     # Designing the interface
-    st.title("Wildfire image crowdsourcing")
-    st.write("The image you upload here will be used to create a public image dataset for wildfire detection.")
+    #st.title("Collecte de photos de feux de forêt")
+    st.title("Améliorez la detection des départs de feux")
+    st.write("La photo que vous partagez ici sera utilisée pour créer un jeu de données public "
+             "pour la detection de feux de forêts.")
+    st.write("Idéalement partagez nous une photo depuis une vue assez dégagée sur une zone forestière, avec et/ou sans flammes/fumées")
+    #st.write("Lorsque vous partagez une photo de départ de feux, n'hésitez pas à "
+    #        "également partager une photo du même point de vue sans fumées ou flammes.")
+
     # Set the columns
     cols = st.columns((2, 1))
-    cols[0].subheader("Input image")
-    cols[1].subheader("Image tags")
+    cols[0].subheader("Selection de la photo")
+    cols[1].subheader("Informations de la photo")
 
     # Sidebar
-    # File selection
-    st.sidebar.title("Image selection")
+    st.sidebar.image("src/static/logo_letters.png")
+
+    # new line
+    st.sidebar.write('\n')
+
+    st.sidebar.markdown("""**A propos de Pyronear** - Pyronear est une association loi 1901, uniquement constituée \
+    de bénévoles, qui a pour objet la préservation des espaces naturels contre les risques d’incendies, \
+    et autres aléas naturels. \n \n Pour cela, nous nous efforçons de trouver des solutions technologiques \
+    performantes, accessibles, open-source et abordables pour la protection contre les risques naturels. """)
+
+    st.sidebar.write('\n')
+    st.sidebar.markdown("""[Site Web](https://pyronear.org/) | [contact](mailto:contact@pyronear.org)""")
```

Comment:
I'm afraid we're gonna get some heavy spam with bots crawling the page. Perhaps say to open an issue on the repo or contact us on twitter/linkedin?

Reply:
You are right ! So I also have to remove contact at pyronear in find out more section. 

I would suggest to remove contact in sidebar and in find out more section, and suggest to contact us via social networks or via our website contact form. What do you think ?

I will handle these points in the PR #10